src/status_im2/contexts/onboarding/create_password/view.cljs

```diff
@@ -1,45 +1,207 @@
 (ns status-im2.contexts.onboarding.create-password.view
-  (:require [quo2.core :as quo]
-            [quo2.foundations.colors :as colors]
-            [react-native.core :as rn]
-            [status-im2.contexts.onboarding.create-password.style :as style]
-            [utils.i18n :as i18n]
-            [utils.security.core :as security]
-            [status-im2.contexts.onboarding.common.background.view :as background]
-            [utils.re-frame :as rf]))
-
-(defn navigation-bar
+  (:require
+    [quo2.core :as quo]
+    [quo2.foundations.colors :as colors]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [status-im2.common.resources :as resources]
+    [status-im2.contexts.onboarding.common.navigation-bar.view :as navigation-bar]
+    [status-im2.contexts.onboarding.create-password.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]
+    [utils.security.core :as security]
+    [utils.string :as utils.string]))
+
+(defn background-image
```

Comment:
hmm, why are you not using the background image in "status-im2.contexts.onboarding.common.background.view" 
Probably you need to adjust that component a bit there instead if the styling is not right.

Reply:
Well, that's a component that still needs some work and since at least on Android it's not looking as expected, I prefered to use the same background I used in the "I'm new to Status" screen.

But it's better to use the right component so that when it gets fixed, it'll be automatically applied to all the screens. I'm using it now. :+1:  thanks for the suggestion.

Reply:
Yep we can fix that in another pr but it will help avoid confusion this way